packages/cli/tests/commonlib/apimValidator.ts

```diff
@@ -362,3 +399,22 @@ class Config {
     return this.config[this.apimPlugin]["apiDocumentPath"];
   }
 }
+
+async function retry<T>(
```

Comment:
>async function retry<T> [](http://example.com/codeflow?start=0&length=23)

As we discussed before, we need to consilidate the `retry` API for all code components in the future.

Reply:
Yes, we need to put the retry handler into a common place for all plugins to use.
The retry handler used in the test is different from the retry handler used in the plugin. 
One is to retry to get the data until it is refreshed or timeout. The other is to retry the operation if the error is thrown.